pkg/dfc/dfc.go

```diff
@@ -914,8 +935,8 @@ func buildImageReference(baseFilename string, tag string, opts Options) string {
 	return newBase
 }
 
-// processRunLine handles the conversion of RUN lines
-func processRunLine(newLine *DockerfileLine, line *DockerfileLine, stagePackages map[int][]string, packageMap PackageMap) {
+// processRunLineWithConverter handles the conversion of RUN lines but supports a RunLineConverter.
```

Comment:
can we modify this method to return an error? then if runLineConverter  returns an error, surface that so that the program exits?